Tools/LyTestTools/ly_test_tools/launchers/launcher_helper.py

```diff
@@ -8,11 +8,18 @@
 """
 
 import logging
+from warnings import warn
 
 import ly_test_tools
 import ly_test_tools._internal.managers.workspace as workspace_manager
 import ly_test_tools.launchers.platforms.base as base_launcher
 
+
+# These are the launchers *currently* supported by the test tools. While other launchers exist, they are not supported
+# by the test tools.
+GAME_LAUNCHERS = ['windows', 'linux', 'android']
+SERVER_LAUNCHERS = ['linux', 'linux_dedicated']
```

Comment:
No windows server launchers? So if we want to run a Windows test we'll need 2 PCs? A Windows and LInux PC?

Reply:
Good catch, fixed.